File: Electronics_Shop_17.Model/Requests/AddUserNotification.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Electronics_Shop_17.Model.Requests
{
    public class AddUserNotification
    {
        public int UserAccountId { get; set; }
        public int NotificationId { get; set; }
        public bool IsRead { get; set; } = false;
    }
}

File: angular17_web/src/app/models/user-notification/add.user.notification.ts
export class AddUserNotification {
  userAccountId!: number;
  notificationId!: number;
  isRead?: boolean;

  constructor(data?: any) {
    if (!data) return;

    this.userAccountId = data.userAccountId;
    this.notificationId = data.notificationId;
    this.isRead = data.isRead;
  }
}
